dts/bindings/tcpc/st,stm32-ucpd.yaml

```diff
@@ -0,0 +1,78 @@
+# Copyright 2021 The Chromium OS Authors
+# SPDX-License-Identifier: Apache-2.0
+
+description: ST STM32 family USB Type-C / Power Delivery.
+             The default values were taken from the
+             LL_UCPD_StructInit function defined in
+             stm32cube/stm32u5xx/drivers/src/stm32u5xx_ll_ucpd.c,
+             stm32cube/stm32g0xx/drivers/src/stm32g0xx_ll_ucpd.c,
+             stm32cube/stm32l5xx/drivers/src/stm32l5xx_ll_ucpd.c, and
+             stm32cube/stm32g4xx/drivers/src/stm32g4xx_ll_ucpd.c.
```

Comment:
nit: use `description: |` for multiline descriptions.

I'd suggest to not reference specific HAL paths (more families may be added in the future, path is not relative to Zephyr, etc.)

```
The default values were taken from the LL_UCPD_StructInit function defined in the HAL.
```

Reply:
done